packages/grape-web-client/demo/iframe.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Embedded Grape Client</title>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=10" />
+  <meta name="viewport" content="width=device-width">
+  <style>
+    html, body {
+      height: 100%;
+      margin: 0;
+    }
+
+    #grape-client {
+      position: absolute;
+      width: 100%;
+      height: 100%;
+    }
+  </style>
+</head>
+<body>
+  <iframe src="https://db2fe37f28fa.ngrok.io/chat/" id="grape-client" />
```

Comment:
this ngrok URL is dead already, maybe put staging or a alpha.chatgrape.com there?